content/ja/snippets/others/delay.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Delay (with await)"
+weight: 10
+ie_support: false
+---
+
+指定した秒数の間待機/スリープを行う delay 関数です。
+
+```js
+function delay(seconds) {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, seconds * 1000);
+  });
+}
+
+return (async () => {
+  // You want to return promise object
+  // so that the test executon engine can wait it to be resolved.
+  // Please write the code inside of this scope.
+
+  // Example:
```

Comment:
And this is an English comment for the Japanese page

Reply:
fixed the loc issue